.github/workflows/links.yml

```diff
@@ -2,6 +2,13 @@ name: Links
 
 on:
   pull_request:
+    paths:
+      - 'concept/**/*.md'
+      - 'concept/**/links.json'
+      - 'docs/*.md'
+      - 'exercises/**/*.md'
+      - 'reference/*.md'
+      - '*.md'
```

Comment:
Perhaps `**/**/*.md` or `**/*.md` could be added as well to cover files other than top-level. Or does `*.md` already cover it?

Reply:
I thought it would, but it doesn't hurt to be more careful. We can always pare it back later.